Makefile

```diff
@@ -25,7 +25,7 @@ clean: ## remove generated byte code, coverage reports, and build artifacts
 	rm -fr dist/
 	rm -fr *.egg-info
 
-test: clean
+test: clean ## Run tests
 	pylint eox_core
-	coverage run --source="." -m pytest
+	coverage run --source="." --omit="*/backends*" -m pytest
```

Comment:
Why are you doing this here instead of using the setup.cfg which already has a coverage section?